qemu/tests/timerdevice_clock_drift_with_ntp.py

```diff
@@ -67,7 +67,7 @@ def _drift_file_exist():
     load_cmd = "for ((I=0; I<`grep 'processor id' /proc/cpuinfo| wc -l`; I++));"
     load_cmd += " do taskset $(( 1 << $I )) /bin/bash -c 'for ((;;)); do X=1; done &';"
     load_cmd += " done"
-    sess_guest_load.sendline(load_cmd)
+    sess_guest_load.cmd(load_cmd)
```

Comment:
Please be aware that you will modify the case execution this way, cause `sendline()` won't wait for the command return, that means below host cpu pin action will be executed while the load_cmd is running. After your modification, it will wait for load_cmd finish before start host actions. I actually doubt it's not what the case wants.

Reply:
'for ((;;)); do X=1; done &', it is run in the background. So the change wont modify the case.

Reply:
@YanhuiMa Right, although the timing is slightly different(only if you add `&` to Line 69 `done` it will be totally same), but that does not seem to affect the case's intention. So Ack.